arangod/Aql/MaterializeExecutor.h

```diff
@@ -112,43 +141,13 @@ class MaterializeExecutor {
   [[nodiscard]] std::tuple<ExecutorState, Stats, AqlCall> produceRows(
       AqlItemBlockInputRange& inputRange, OutputAqlItemRow& output);
 
-  /**
-   * @brief skip the next Row of Aql Values.
-   *
-   * @return ExecutorState, the stats, and a new Call that needs to be send to
-   * upstream
-   */
-  [[nodiscard]] std::tuple<ExecutorState, Stats, size_t, AqlCall> skipRowsRange(
-      AqlItemBlockInputRange& inputRange, AqlCall& call);
-
-  void initializeCursor() noexcept {
-    // Does nothing but prevents the whole executor to be re-created.
-    _collection = {};
-  }
-
```

Comment:
Why it was removed? It looks important for materialize via multiget

Reply:
How can it be important for the search case, when it's not used in the code.

The `initializeCursor` is called for each subquery start to reset the executor. If it is not implemented, the executor is simply rebuilt in place.

Reply:
Yes, but we don't need to rebuild this executor, it's rebuilding free already allocated memory and resources, and then allocate them another time